notebooks/beir_cqadupstack.ipynb

```diff
@@ -0,0 +1,546 @@
+    "stats = RawBeirDataset(\"stats\", test = True)\n",
+    "tex = RawBeirDataset(\"tex\", test = True)\n",
+    "unix = RawBeirDataset(\"unix\",test = True)\n",
+    "webmasters = RawBeirDataset(\"webmasters\", test = True)\n",
+    "wordpress = RawBeirDataset(\"wordpress\", test = True)\n",
+    "\n",
+    "download_location = \"datasets\"\n",
+    "desired_datasets = RawBeirDatasets(datasets=[android,english,gaming,gis,mathematica,physics,programmers,stats,tex,unix,webmasters,wordpress],\n",
+    "                                   download_location=download_location)\n",
+    "\n",
+    "# If download_location is set to None the data should not re-download, instead it will load from default location\n",
+    "# desired_datasets = RawBeirDatasets([android,english,gaming,gis,mathematica,physics,programmers,stats,tex,unix,webmasters,wordpress]\", None)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "jo_45suJzxkX",
+   "metadata": {
+    "id": "jo_45suJzxkX",
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Load and split raw data\n",
+    "Once we have all the desired datasets defined we can load them, load_BEIR_datasets downloads (if necessary), loads, and prepares the raw data from the selected datasets.\n",
+    "\n",
+    "It returns three values:\n",
+    "raw_train_data, raw_dev_data, raw_test_data,\n",
+    "but as the train and dev subsets are not present in these datasets, these can be ignored.\n",
+    "\n",
+    "To get the train and dev(validation) subsets, we use the split_BEIR_dataset to split the original test data.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8f5a0b15-d2ee-405e-81be-64c627a02454",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "8f5a0b15-d2ee-405e-81be-64c627a02454",
+    "outputId": "dd471733-f0eb-43a1-f4ca-439a84caf99c",
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 22998/22998 [00:00<00:00, 242199.02it/s]\n",
+      "100%|██████████| 40221/40221 [00:00<00:00, 227932.64it/s]\n",
+      "100%|██████████| 45301/45301 [00:00<00:00, 271704.81it/s]\n",
+      "100%|██████████| 37637/37637 [00:00<00:00, 203171.78it/s]\n",
+      "100%|██████████| 16705/16705 [00:00<00:00, 193812.80it/s]\n",
+      "100%|██████████| 38316/38316 [00:00<00:00, 192532.68it/s]\n",
+      "100%|██████████| 32176/32176 [00:00<00:00, 208979.33it/s]\n",
+      "100%|██████████| 42269/42269 [00:00<00:00, 201072.49it/s]\n",
+      "100%|██████████| 68184/68184 [00:00<00:00, 165782.21it/s]\n",
+      "100%|██████████| 47382/47382 [00:00<00:00, 214092.20it/s]\n",
+      "100%|██████████| 17405/17405 [00:00<00:00, 239810.33it/s]\n",
+      "100%|██████████| 48605/48605 [00:00<00:00, 198100.41it/s]\n"
+     ]
+    }
```

Comment:
```suggestion
```

Reply:
todo

Reply:
should be done

Reply:
@MarekToma Well done. Note that you should be able to directly commit my suggestions to Git just by clicking a button. Also, instead of writing "should be done", feel free to mark the conversation as resolved.